### DISCUSSION TITLE
11/30 Webinar recording and transcript

### DISCUSSION THREAD

**Discussion by kavkuk** [CMSgov/price-transparency-guide#296] [Q&A] [answered]
@shaselton-usds Will you be posting the recording and transcript of 11/30 webinar? I do not see it here: https://www.cms.gov/healthplan-price-transparency/resources/webinars

**Comment by tparency** [comment, marked as answer]
The replay, transcript and presentation slides are now on the site. https://www.cms.gov/healthplan-price-transparency/resources/webinars

**Reply by shaselton-usds** [reply]
thanks for answering.
